Sources/BraintreeDropIn/BTCardFormViewController.m

```diff
@@ -285,6 +297,11 @@ - (void)updateRequiredFields {
     }
 }
 
+- (void) tappedPrivacyPolicy: (UITapGestureRecognizer *) gestureRecognizer {
+    NSURL *privacyPolicy = [NSURL URLWithString:@"https://www.paypal.com/us/legalhub/home?locale.x=en_US"];
```

Comment:
Same comment as I had on the web PR [here](https://github.com/braintree/braintree-web-drop-in/pull/899/files#r1275042361)

Reply:
Updated to remove the locale, good call: https://github.com/braintree/braintree-ios-drop-in/pull/420/commits/619c4ae1500d89ae598b4d687037004ee994b98c